operator/cmd/mesh/uninstall.go

```diff
@@ -250,7 +250,7 @@ func preCheckWarnings(cmd *cobra.Command, kubeClient kube.CLIClient, uiArgs *uni
 			message += fmt.Sprintf(GatewaysRemovedWarning, gwList)
 		}
 	}
-	if uiArgs.skipConfirmation {
+	if dryRun || uiArgs.skipConfirmation {
```

Comment:
Is it really dry running? It seems like it will just print the message and continue with the run.

Reply:
Dryrun itself is functioning well. This is simply to bypass confirmation in `--dry-run` mode - which is no need in dryrun, similar to other istioctl commands.

Reply:
I see. Thanks.